commands/login.go

```diff
@@ -116,7 +110,7 @@ func validateLogin(gatewayURL string, user string, pass string) error {
 	req.SetBasicAuth(user, pass)
 	res, err := client.Do(req)
 	if err != nil {
-		return fmt.Errorf("cannot connect to OpenFaaS on URL: %s", gatewayURL)
+		return fmt.Errorf("cannot connect to OpenFaaS on URL: %s. %v", gatewayURL, err)
```

Comment:
helpful to print the error, so user can solve problem. i.e. certificate error, timeout etc...